hacks/update-store-json.ts

```diff
@@ -0,0 +1,87 @@
+import ArweaveSDK from '../src/modules/arweave/client';
+import singletons from '../src/modules/singletons';
+import { ajvParse } from '../src/modules/utils/json';
+import { SCHEMAS } from '../src/modules/singletons/json-schemas';
+import { resultThenAsync } from '../src/modules/utils/result';
+import {
+  parseNotarized,
+  unpackNotarized,
+  verifyNaclSelfContained,
+} from '../src/modules/notary/server';
+import { PublicKey } from '@solana/web3.js';
+import { formatMessage } from '../src/modules/storefront/put-storefront';
+import type { Storefront } from '../src/modules/storefront/types';
+import { stylesheet } from '../src/modules/theme';
+import fs from 'fs';
+
+const storefrontJSON = fs.readFileSync('./hacks/storefront.json', 'utf-8');
+
+console.log('storefront JSON:', storefrontJSON);
+
+const verifyPutParams = async (params: any) => {
+  const schemas = singletons.jsonSchemas;
+  const parseStorefront = ajvParse(schemas.parser(SCHEMAS.storefront));
+
+  const payloadRes = await resultThenAsync(parseNotarized<Storefront>(params), (params) =>
+    unpackNotarized(
+      params,
+      verifyNaclSelfContained((s) => new PublicKey(s.pubkey).toBuffer()),
+      { parse: parseStorefront, format: formatMessage }
+    )
+  );
+
```

Comment:
We don't need the notarize these since ran by us. This is for publically exposed uploads.